src/darwin/Framework/CHIP/templates/CHIPStructsObjc-src.zapt

```diff
@@ -19,6 +19,23 @@ NS_ASSUME_NONNULL_BEGIN
 @end
 
 {{/zcl_structs}}
+
+{{#zcl_events}}
+@implementation CHIP{{asUpperCamelCase parent.name}}Cluster{{asUpperCamelCase name}}Event
+- (instancetype)init
+{
+  if (self = [super init]) {
+    _priorityLevel = @(0);
```

Comment:
Where do we use `priorityLevel`?  It looks unused at first glance...

Reply:
Left over from development - will remove.